polaris/dataset/_adapters.py

```diff
@@ -13,10 +15,10 @@ class Adapter(Enum):
         BYTES_TO_MOL: Convert a RDKit binary string to a RDKit molecule.
     """
 
-    SMILES_TO_MOL = dm.to_mol
-    BYTES_TO_MOL = dm.Mol
+    SMILES_TO_MOL = "SMILES_TO_MOL"
+    BYTES_TO_MOL = "BYTES_TO_MOL"
```

Comment:
Minor, but since the `name` and the `value` of the Enum are the same, maybe we should just use [`auto()`](https://docs.python.org/3/library/enum.html#enum.auto) as the value to prevent confusion on whether to use the `name` or the `value`.

Reply:
To be really safe, maybe we combine it with the [`@unique`](https://docs.python.org/3/library/enum.html#enum.unique) decorator.